content/v2.0/query-data/flux/calculate-percentages.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Calculate percentages with Flux
+list_title: Calculate percentages
+description: >
+  Use [`pivot()` or `join()`](/v2.0/query-data/flux/calculate-percentages/#pivot-vs-join)
+  and the [`map()` function](/v2.0/reference/flux/stdlib/built-in/transformations/map/)
+  to align operand values into row-wise sets and calculate a percentage.
+menu:
+  v2_0:
+    name: Calculate percentages
+    parent: Query with Flux
+weight: 220
+aliases:
+ - /v2.0/query-data/guides/manipulate-timestamps/
+related:
+  - /v2.0/query-data/flux/mathematic-operations
+  - /v2.0/reference/flux/stdlib/built-in/transformations/map
+  - /v2.0/reference/flux/stdlib/built-in/transformations/pivot
+  - /v2.0/reference/flux/stdlib/built-in/transformations/join
+list_query_example: percentages
+---
+
+
+Calculating percentages using multiple values in a queried data set is a common use case for time series data.
+With Flux, all operand values need to exists in a single row to use them in a mathematic calculation.
+Once operands are aligned in rows, use `map()` to re-map values in the row and calculate a percentage.
+
+**To calculate percentages:**
+
+1. Query the necessary operand values.
+2. Use [`pivot()` or `join()`](#pivot-vs-join) to align operand values into row-wise sets.
+3. Use [`map()`](/v2.0/reference/flux/stdlib/built-in/transformations/map/)
+   to divide the numerator operand value by the denominator operand value and multiply by 100.
+
+{{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[pivot()](#)
```

Comment:
It might be nice to include all examples up here at the top, and then reword to focus on user's data (rather than pivot or join): 

### Examples of calculating percentages:
-  from multiple data sources
- from two buckets
- from two measurements
- from 2 fields in one measurement
- from 3 fields in one measurement

Reply:
I move the note about using pivot vs join before this code block and simplified it to just use point (no code tabs)